rotkehlchen/tests/unit/accounting/test_prefork_assets.py

```diff
@@ -140,18 +142,19 @@ def test_buying_selling_btc_before_bchfork(accountant, google_service):
     assert len(buys) == 1
     assert buys[0].remaining_amount == amount_bch
     assert buys[0].timestamp == 1491593374
-    assert buys[0].rate.is_close('1128.98961538')
+    assert buys[0].rate.is_close('1128.905')
```

Comment:
Why did this and the other numbers in the test change? With inlclude in cost basis being False, the result should be same as before?

The change is super super small, so sounds tricky. What changed? Was it doing some double counting of fees before?